docs/src/main/asciidoc/reactive-sql-clients.adoc

```diff
@@ -31,21 +31,34 @@ Feedback is welcome on our https://groups.google.com/d/forum/quarkus-dev[mailing
 
 In this guide, you will learn how to implement a simple CRUD application exposing data stored in *PostgreSQL* over a RESTful API.
 
+The code presented is available in this {quickstarts-base-url}[GitHub repository] under the link:{quickstarts-tree-url}/getting-started-reactive-crud[`getting-started-reactive-crud` directory].
```

Comment:
Could you mimick what we do in the other guides instead?

Typically https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/validation.adoc?plain=1#L30-L37 .

We try to have the same structure for all guides.

Now sometimes it doesn't make sense so if you think it doesn't make sense for you, please push back.

Reply:
Will do, thanks for the feedback

Reply:
Done